src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -347,60 +345,56 @@ private EmailWrapper generateSessionLinksRecoveryEmailForExistingStudent(String
                                                                              List<StudentAttributes> studentsForEmail) {
         String emailBody;
 
-        Instant endTime = Instant.now();
-        Instant startTime = endTime.minus(SESSION_LINK_RECOVERY_DURATION);
+        Instant searchStartTime = TimeHelper.getInstantDaysOffsetBeforeNow(SESSION_LINK_RECOVERY_DURATION);
```

Comment:
I don't think you need a new method. `SESSION_LINK_RECOVERY_DURATION` is in days, and there is already a method which caters to exactly that.

Reply:
Wasn't aware of that method, possibly because it was added about the same time as the other method. Regardless, will switch over to the existing one.